source/js/openstreetmap/showPost.js

```diff
@@ -30,51 +46,63 @@ class ShowPost {
         });
     }
 
+    handleParams() {
```

Comment:
Could we give this function a less generic name, possibly extract it into smaller functions?